include/vara/Feature/FeatureSourceRange.h

```diff
@@ -101,6 +101,8 @@ class FeatureSourceRange {
       return StrS.str();
     }
 
+    [[nodiscard]] size_t nested() const { return Class.size(); }
```

Comment:
Hmm, `nested` sounds like a `true/false` kind check. Maybe something like `nestingDepth`?

Reply:
And could you please write a small method doc to the function that explains what is returned.